uaclient/config.py

```diff
@@ -259,6 +259,11 @@ def _status(self) -> 'Dict[str, Any]':
                 'name': ent.name, 'entitled': contract_status,
                 'status': op_status, 'statusDetails': op_details}
             response['services'].append(service_status)
+        support = self.entitlements.get('support', {}).get('entitlement')
+        if support:
+            response['techSupportLevel'] = support.get(
+                'affordances', {}).get('supportLevel', status.INAPPLICABLE)
```

Comment:
This leaves us with `status.INAPPLICABLE` as the default in two different places (here and in `DEFAULT_STATUS`); can we refactor things so that if/when this default changes (looking at you #501) we don't have to ensure that both are kept in sync?

Reply:
Using the DEFAULT_STATUS['techicalSupportLevel'] we have defined... and will grab #501 next